Doc/whatsnew/3.15.rst

```diff
@@ -850,6 +852,93 @@ csv
   (Contributed by Maurycy Pawłowski-Wieroński in :gh:`137628`.)
 
 
+.. _whatsnew315-jit:
+
+Upgraded JIT Compiler
+=====================
+
+The JIT compiler introduced in
+`Python 3.13 <https://docs.python.org/3.15/whatsnew/3.13.html#an-experimental-just-in-time-jit-compiler>`_
+has been significantly upgraded.
+
+Current results from the pyperformance benchmark suite report
+`3-4% <https://github.com/facebookexperimental/free-threading-benchmarking/blob/main/results/bm-20251214-3.15.0a2%2B-6cddf04-JIT/bm-20251214-vultr-x86_64-python-6cddf04344a1e8ca9df5-3.15.0a2%2B-6cddf04-vs-base.svg>`_
+geometric mean performance over the standard CPython interpreter built with
+all optimizations enabled. The speedups range from roughly 20% slowdown to
+over 100% speedup (ignoring the ``unpack_sequence`` microbenchmark) on the
+x86-64 Linux and AArch64 macOS systems. These results are not yet final.
```

Comment:
Is this -20% - +100% with JIT vs no JIT on the same CPython commit, or versus a previous CPython with the JIT enabled?